src/test/java/timetables_sort_use_case/interface_adapters/TestAllTimetablesView.java

```diff
@@ -0,0 +1,17 @@
+package timetables_sort_use_case.interface_adapters;
+
+import display_timetable_use_case.frameworks_and_drivers.TimetableViewModel;
+
+/**
+ * A test view that mocks up AllTimetablesView.
+ * timetables is the TimetableViewModel array passed to the view.
+ */
+public class TestAllTimetablesView implements AllTimetablesView {
```

Comment:
You might be able to just have this be an anonymous class instead in future.
The syntax is something along the lines of

AllTimetablesView view = new AllTimestablesView(){
// Override the methods in here to mock up the view.

}

Reply:
Oh :(. WIth an anonymous class you can override to the new functions right? Meh. I guess if it breaks things it's not something worth taking a risk on.